bin/notifications.sh

```diff
@@ -458,12 +445,12 @@ if [[ "${email}" == 'true' ]]; then
 else
 	log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' 'Email option option not enabled: will log for debugging on stdout.'
 fi
-if [[ -z "${selectedState:-}" ]]
+if [[ -z "${selectedPhaseState:-}" ]]
 then
-	selectedState='all'
-	log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' 'selectedState=all'
+	selectedPhaseState='all'
```

Comment:
If you move `selectedPhaseState='all'` to `declare selectedPhaseState='all'` just after line `declare email='false'` above, you do not need if then else here and can simply log `log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "selectedPhaseState=${selectedPhaseState}"`

Reply:
check